Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity/CredentialFeatureIDInterceptor.swift

```diff
@@ -0,0 +1,42 @@
+//
```

Comment:
This interceptor is used by `IdentityProvidingXXXClient` structs codegenerated for each internal service client modules (`InternalAWSTS`, `InternalAWSSSO`, and `InternalAWSSSOOIDC`) to inject feature IDs into operation context. The injected feature ID list is retrieved by `BusinessMetrics` when constructing the user agent.